RussianFilter/sections/specific.txt

```diff
@@ -49,6 +49,7 @@ crazymailing.com,lotzon.pro,novosti-n.org,prokazan.ru,20minut.ua,alnews.com.ua,g
 ! vast ads on anime sites - for example: animego.org, yummyanime.club, etc
 /:\/\/[0-9]?[a-z0-9]{11,}(\.com|\.in\.net)/$xmlhttprequest,domain=aniqit.com|kodik.info|start.u-cdn.top|ustore.bz|u-stream.in
 !
+rus-ege.sdamgia.ru##div[style="width:160px;height: 205px;margin: 0 auto;"]
```

Comment:
I guess it's ok, but we need to use it in other subdomains, so `sdamgia.ru##div[style="width:160px;height: 205px;margin: 0 auto;"]`.